uint8 class_pred
uint32 x1
uint32 y1
uint32 x2
uint32 y2
float32 score
string label
